src/olympia/devhub/templates/devhub/addons/submit/source.html

```diff
@@ -63,6 +63,12 @@ <h3>{{ _("You must submit your extension's source code.") }}</h3>
                 Source files (aside from open-source third party libraries) cannot be transpiled, concatenated, minified, or otherwise machine-generated.
             {% endtrans %}
         </p>
+        <p>
+            {% trans %}
+            The maximum file size accepted is 
+            {% endtrans %}
+            {{ max_upload_size|filesizeformat }}.
+        </p>
```

Comment:
See below, the variable needs to be passed to the `{% trans %}` block. You can do `{% trans max_size=max_upload_size|filesizeformat %}The maximum file size accepted is {{ max_size }}. {% endtrans %}`.